lib/flutter_calendar_carousel.dart

```diff
@@ -964,12 +965,28 @@ class _CalendarState<T> extends State<CalendarCarousel<T>> {
             );
           }
         } else {
-          if (widget.markedDateWidget != null) {
-            tmp.add(widget.markedDateWidget);
-          } else {
-            tmp.add(defaultMarkedDateWidget);
+
```

Comment:
Why empty lines here?

Reply:
Just for readability, feel free to remove.

Reply:
Ok lets leave it there for now.